### PR TITLE
Add tutorial to FoodChain

### PR DESCRIPTION
**Pull request by AmiMappy**
Added tutorial functionality to FoodChain activity.

### CODE REVIEW

File: activities/FoodChain.activity/index.html [hunk 1]
```diff
@@ -36,6 +36,13 @@
 	<script src="app.js" type="text/javascript"></script>
     <script>if (window.module) module = window.module;</script>
 	<link href="styles.css" rel="stylesheet" type="text/css" />
+	<!-- Tutorial -->
+	<link rel="stylesheet" href="bootstrap-tour-standalone.min.css">
+	<script>if (typeof module === 'object') {window.module = module; module = undefined;}</script>
+	<script src="lib/jquery.min.js"></script>
+	<script src="lib/bootstrap-tour-standalone.min.js"></script>
+	<!-- <script>if (window.module) module = window.module;</script> This line already exists-->
```
Comment (llaske) [inline activities/FoodChain.activity/index.html]: Should not be commented, inclusion of JS libraries should be surrounded by the two lines mentioned.

File: activities/FoodChain.activity/lib/onyx/package.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
 enyo.depends(
 	// $lib/onxy loads the debug code for now, probably it should load the built code as below
-	"source"
-	/*
-	"build/onyx.css",
-	"build/onyx.js"
-	*/
+	"source",
+	"bootstrap-tour-standalone.min.css",
```
Comment (llaske) [inline activities/FoodChain.activity/lib/onyx/package.js]: You should not update this file, including CSS and JS files in the index.html is sufficient

**Comment by AmiMappy** [comment]
Uncaught Error: Mismatched anonymous define() module: function(o){return t.Tour=e(o)}
http://requirejs.org/docs/errors.html#mismatch
    at makeError (require.js:159)
    at intakeDefines (require.js:1192)
    at Object.localRequire [as require] (require.js:1366)
    at requirejs (require.js:1691)
    at Function.req.config (require.js:1699)
    at loader.js:1
makeError @ require.js:159
intakeDefines @ require.js:1192
localRequire @ require.js:1366
requirejs @ require.js:1691
req.config @ require.js:1699
(anonymous) @ loader.js:1

The above error disappears after reloading page couple of times.

**Comment by AndreaGon** [comment]
Hello @AmiMappy ! I think you should also add the svg files. These will be used as images for the help button, cancel, next and previous buttons.

**Comment by AmiMappy** [comment]
I tried to add a tutorial pop-up for the cards, but I just saw a bunch of pop-ups appear in the home view instead.

File: activities/FoodChain.activity/lib/onyx/package.js [hunk 3]
```diff
@@ -1,8 +1,4 @@
 enyo.depends(
 	// $lib/onxy loads the debug code for now, probably it should load the built code as below
 	"source"
-	/*
```
Comment (llaske) [inline activities/FoodChain.activity/lib/onyx/package.js]: You don't need to change this file

File: activities/FoodChain.activity/util.js [hunk 4]
```diff
@@ -60,7 +60,8 @@ FoodChain.setLocale = function() {
 	else if (l10n.language.code.indexOf("pt_BR") == 0) { document.getElementById("pt_BR-button").classList.add('active'); }
 	if (FoodChain.context.object != null)
 		FoodChain.context.object.setLocale();
-}
+};
```
Comment (llaske) [inline activities/FoodChain.activity/util.js]: You don't need to change this file

**Comment by AmiMappy** [comment]
fixed
The mentioned error has been fixed. Working on the adapting...

**Comment by AmiMappy** [comment]
adapt
The adaptation is also done.

File: activities/FoodChain.activity/index.html [hunk 5]
```diff
@@ -34,8 +31,12 @@
 	<script src="network.js" type="text/javascript"></script>
 	<script src="credits.js" type="text/javascript"></script>
 	<script src="app.js" type="text/javascript"></script>
-    <script>if (window.module) module = window.module;</script>
```
Comment (llaske) [inline activities/FoodChain.activity/index.html]: All JS file should be around these lines. You should not remove it.

File: activities/FoodChain.activity/js/activity.js [hunk 6]
```diff
@@ -28,6 +28,7 @@ define(["sugar-web/activity/activity","webL10n","sugar-web/graphics/radiobuttons
 			FoodChain.setLocale();
 		};
 
+
```
Comment (llaske) [inline activities/FoodChain.activity/js/activity.js]: Do not change line not concerned by your update.

File: activities/FoodChain.activity/lib/onyx/package.js (hunk 3 above)
Comment (llaske) [reply activities/FoodChain.activity/lib/onyx/package.js]: This package should not be changed

File: activities/FoodChain.activity/lib/tutorial.js [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+				placement: "bottom",
+				title: l10n.get("TutoEnTitle"),
+				content: l10n.get("TutoEnContent")
+			},
+			{
+				element: "#fr-button",
+				title: l10n.get("TutoFrTitle"),
+				content: l10n.get("TutoFrContent")
+            },
+            {
```
Comment (llaske) [inline activities/FoodChain.activity/lib/tutorial.js]: Don't mix indentation with TAB and indentation with white space

**Comment by AmiMappy** [comment]
works

I am not getting the error.

-----------------------------------------------------

alright

Also, the indentation seems fine.

**Comment by llaske** [comment]
Your IDE probably don't differentiate indentation.

image

Regarding the error message, try to clean your browser cache.

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/pull/545

**Pull request closed without merging**